synapse/storage/databases/main/events_worker.py

```diff
@@ -1408,10 +1408,10 @@ def get_all_new_forward_event_rows(
         ) -> List[Tuple[int, str, str, str, str, str, str, str, str]]:
             sql = (
                 "SELECT e.stream_ordering, e.event_id, e.room_id, e.type,"
-                " state_key, redacts, relates_to_id, membership, rejections.reason IS NOT NULL"
+                " se.state_key, redacts, relates_to_id, membership, rejections.reason IS NOT NULL"
                 " FROM events AS e"
                 " LEFT JOIN redactions USING (event_id)"
-                " LEFT JOIN state_events USING (event_id)"
+                " LEFT JOIN state_events se USING (event_id)"
```

Comment:
```suggestion
                " LEFT JOIN state_events AS se USING (event_id)"
```
(personal preference, though it is also used above)